requirements.txt

```diff
@@ -12,12 +12,12 @@ mwtypes >= 0.2.0, < 0.3.999
 mwparserfromhell >= 0.5.1, < 0.5.999
 mysqltsv >= 0.0.7, < 0.0.999
 nltk >= 3.0.0, < 3.0.999
-numpy >= 1.14.0, < 1.14.999
+numpy >= 1.16.2, < 2.0.999
```

Comment:
I think we'll want to stay within minor version for these ranges.  E.g. `numpy >= 1.16.2, < 1.6.999`.  See https://semver.org/ for the meaning of these numbers.

Reply:
cool. Will apply the review made :)

Reply:
In here `E.g. numpy >= 1.16.2, < 1.6.999` it should be  <1.16.999 right ?